src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -151,6 +151,55 @@ void CascadiaSettings::_CreateDefaultProfiles()
     _profiles.emplace_back(powershellProfile);
     _profiles.emplace_back(cmdProfile);
     
+    // Add all WSL distros from: HKCU\Software\Microsoft\Windows\CurrentVersion\Lxss
+    Profile WSLDistro{};
+    HKEY hLXSSKey;
+    // Should I use RegOpenCurrentUser to access the uncached profile?

```

Comment:
I can find no reason to not use RegOpenCurrentUser, but perhaps one of the console team members knows more?

Reply:
I updated it to include this function.  It won't be difficult to change it back.  I think it makes sense to use a live copy over a cached copy.